src/__tests__/assets/v1/v1.0.0/gnosis_safe.json

```
@@ -3,7 +3,9 @@
   "released": true,
   "contractName": "GnosisSafe",
   "version": "1.0.0",
-  "codeHash": "0xe1f1593df76e69abc2d692792c80f329457551d5e83dde597546a1d58764da80",
+  "codeHash": {
+    "canonical": "0xe1f1593df76e69abc2d692792c80f329457551d5e83dde597546a1d58764da80"
+  },
   "addresses": {
     "canonical": "0xb6029EA3B2c51D09a50B53CA8012FeEB05bDa35A"
   },
```

nit: I would change this. This is like representing an array of structs as arrays of fields. Something like:

```suggestion
  "deployments": {
    "canonical": {
      "address": "0xb6029EA3B2c51D09a50B53CA8012FeEB05bDa35A",
      "codeHash": "0xe1f1593df76e69abc2d692792c80f329457551d5e83dde597546a1d58764da80"
    }
  },
```

---

I'm ok with this change if @mmv08 doesn't have a problem 👍🏾